crates/payload/builder/src/error.rs

```diff
@@ -6,6 +6,9 @@ use reth_transaction_pool::BlobStoreError;
 use revm_primitives::EVMError;
 use tokio::sync::oneshot;
 
+#[cfg(feature = "optimism")]
+use revm_primitives::Address;
```

Comment:
looks like we're only using this once, please use full path instead

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/149